docusaurus.config.js

```diff
@@ -35,6 +35,8 @@ const config = {
         docs: {
           routeBasePath: '/',
           sidebarPath: require.resolve('./sidebars.js'),
+          editUrl:
+            'https://github.com/keptn-sandbox/new-keptn-docs-engine/edit/main/',
```

Comment:
I think we will need to figure out how to enable this button on pages which aren't generated using CI and shouldn't be edited by hand.

Reply:
https://github.com/keptn-sandbox/new-keptn-docs-engine/issues/22